### PR TITLE
ci: Included githubactions in the dependabot config

### PR DESCRIPTION
**Pull request by naveensrinivasan**
This should help with keeping the GitHub actions updated on new releases. This will also help with keeping it secure.

Dependabot helps in keeping the supply chain secure https://docs.github.com/en/code-security/dependabot

GitHub actions up to date https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@nixpanic 👀

**Review by nixpanic — DISMISSED** [review]
great, thanks again!

**Comment by Madhu-1** [comment]
@naveensrinivasan Thanks for the contribution, can you please fix CI failures?

**Comment by humblec** [comment]
@naveensrinivasan ```
https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>
✖   body's lines must not be longer than 80 characters [body-max-line-length]```

**Comment by nixpanic** [comment]
CI failures are addressed with https://github.com/nixpanic/ceph-csi/commit/49b84bbe0d0d01d186e58bef0800d65b80926bf8

@naveensrinivasan, let me know if you want me to take this forward. Thanks!

**Comment by naveensrinivasan** [comment]
> CI failures are addressed with [nixpanic@49b84bb](https://github.com/nixpanic/ceph-csi/commit/49b84bbe0d0d01d186e58bef0800d65b80926bf8)
> 
> @naveensrinivasan, let me know if you want me to take this forward. Thanks!

I have pushed the changes for commit message length.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -47,3 +47,12 @@ updates:
       - ci/skip/e2e
     commit-message:
       prefix: "rebase"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule: 
+      interval: weekly
+    rebase-strategy: disabled
+    labels: 
+     - "rebase"
```
Comment (nixpanic) [inline .github/dependabot.yml:56]: this probably needs an extra indention?

**Review by nixpanic — APPROVED** [review]

**Review by yati1998 — CHANGES_REQUESTED** [review]
Please check the commitlint and extralint errors.

**Comment by naveensrinivasan** [comment]
> CI failures are addressed with [nixpanic@49b84bb](https://github.com/nixpanic/ceph-csi/commit/49b84bbe0d0d01d186e58bef0800d65b80926bf8)
> 
> @naveensrinivasan, let me know if you want me to take this forward. Thanks!

@nixpanic  I would appreciate if you can please take care of this.

**Comment by nixpanic** [comment]
> @nixpanic I would appreciate if you can please take care of this.

Sure! #3084 is a replacement for this.

**Pull request closed without merging**